connectors/src/lib/models/microsoft.ts

```diff
@@ -57,12 +69,12 @@ MicrosoftConfiguration.init(
   }
 );
 
-ConnectorModel.hasMany(MicrosoftConfiguration);
+ConnectorModel.hasMany(MicrosoftConfigurationModel);
 
 // MicrosoftConfigurationRoot stores the drive/folders/channels selected by the user to sync.
-export class MicrosoftConfigurationRoot extends Model<
-  InferAttributes<MicrosoftConfigurationRoot>,
-  InferCreationAttributes<MicrosoftConfigurationRoot>
+export class MicrosoftConfigurationRootModel extends Model<
```

Comment:
And here not sure we should call that "ConfigurationRoot", a bit unclear what a configurationroot is
Maybe just "Root" or "contentroot" or "noderoot"?